### ISSUE TITLE
WhatsApp show notifications in silent mode.....

### ISSUE THREAD

**Issue by ducarpit** [meetfranz/franz#460]
Hi,
i use WhatsApp, Telegram and Slack in my Franz app.
I disabled "Enable notifications" in my setting of WhatsApp but sometimes Franz show them although i disable this function. Then, if i close and reopen the app it works fine.
I use Windows 10 64bit.
Franz version: 5.0.0-beta.14
I hope this warning will help you.
Christian

**Comment by adlk** [comment]
What confuses me is that that it _sometimes_ shows the notification – not always unless you restart?

**Comment by ducarpit** [comment]
No, for example this morning i powered on my PC then i opened Franz and it works fine (it doesn't show WhatsApp notification). Maybe, tomorrow morning i will open Franz and it will show unexpectly Whatsapp notification.....it's random!
The same thing appen with Telegram.......
Christian

**Comment by adlk** [comment]
Umm... this sounds too crazy 🙃
This seems to be related to #246

**Comment by ducarpit** [comment]
Maybe....if you need more information i will help you.  Or you can change "more informatione required" status....

**Comment by adlk** [comment]
Thanks for the offer. For now, it is crucial to find a way to find a way to reproduce the issue.

**Comment by haraldox** [comment]
@ducarpit do you also hear notification sounds? did you disable notfications globally in Franz on the sidebar?

**Comment by ducarpit** [comment]
@haraldbox Yes, but it happen sometimes....not always!

**Comment by Duartemartins** [comment]
Can confirm I also get whatsapp notifications despite having disabled notifications for the channel and disabled notifications globally. This is on a mac 10.13.3. The difference for me is that it never works even when restarting.
